editor/project_manager.cpp

```diff
@@ -1818,8 +1820,9 @@ void ProjectList::_panel_input(const Ref<InputEvent> &p_ev, Node *p_hb) {
 
 		emit_signal(SNAME(SIGNAL_SELECTION_CHANGED));
 
-		if (!mb->is_ctrl_pressed() && mb->is_double_click()) {
+		if (!mb->is_ctrl_pressed() && mb->is_double_click() && !_project_opening_initiated) {
```

Comment:
Since it's not immediately obvious as to why this is done, there should be a comment:

```suggestion
		// Do not allow opening a project more than once using a single project manager instance.
		// Opening the same project in several editor instances at once can lead to various issues.
		if (!mb->is_ctrl_pressed() && mb->is_double_click() && !_project_opening_initiated) {
```

Reply:
Thanks for the suggestion! I've added the comment now.